### DISCUSSION TITLE
"Contact List App" Laravel Livewire 3 installed on Sail, with Breeze.

### DISCUSSION THREAD

**Discussion by iamivanabella** [livewire/livewire#8606] [1. Help]
I created a simple project "Contact List App"

However when I want to edit a contact, it doesn't show any information not until I refresh the page. I also just notice even the save button isn't working not until I refresh the page. Can anyone help me?

![test](https://github.com/livewire/livewire/assets/72548411/2ae538f1-623d-40d6-9c98-7ca82b7dba28)

**Comment by iamivanabella** [comment]
```
<a href="/contact/edit/{{ $contact->id }}">Edit</a>
<x-secondary-button class="my-1" wire:navigate :href="route('edit-contact', $contact)" >{{ __('Edit') }}</x-secondary-button>

```
So i guess the issue here is wire navigate, on the a tag it works fine.

**Reply by lrljoe** [reply]
Your problem is almost certainly with the code for your blade component.

Livewire absolutely supports blade components 

So let's dive into the issue:

```<x-secondary-button>```

What's the content of the blade for that component?

Why are you passing href as a property?  
Are you outputting all passed attributes within your blade component?

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and we can't replicate this with the infromation give so I'm going to close this. If you're still experiencing problems, please open a new discussion and provide code examples that demonstrate the issue that we can copy and paste so we can investigate. Thanks!
